doc/source/bibtex.rst

```diff
@@ -11,11 +11,14 @@ in :ref:`bibtex-options`.
 Unicode output
 --------------
 
-Depending on the TeX engine used, you may be able to use unicode characters in
-your BibTeX files, such as with XeTeX or LuaTeX. In that case there is no need
-to write ``\`{a}`` instead of ``à`` in words such as ``apareixerà``. To control
-the output of unicode characters in Papis, use the :confval:`bibtex-unicode`
-setting.
+The setting :confval:`bibtex-unicode` can be used to allow unicode characters in
+documents.
```

Comment:
This was my own addition, because the rest of the section needs to tie to *something* :smile: 

Let me know if this should be reworded, I added it a bit lazily.